src/common/user-store/user-store.ts

```diff
@@ -109,7 +116,29 @@ export class UserStore extends BaseStore<UserStoreModel> /* implements UserStore
       });
     }, {
       fireImmediately: true,
-    });
+    }); 
+  }
+
+  // Returns monaco editor options for selected editor type (the place, where a particular instance of the editor is mounted)
+  getEditorOptions(type: EditorType): monaco.editor.IStandaloneEditorConstructionOptions{
+    return {
+      automaticLayout: true,
+      tabSize: this.editorConfiguration.tabSize ?? defaultEditorConfig.get(type).tabSize,
+      minimap: this.editorConfiguration.miniMap ?? defaultEditorConfig.get(type).miniMap,
+      lineNumbers: this.editorConfiguration.lineNumbers
+    };
+  }
+
+  setLineNumbers(lineNumbers: monaco.editor.LineNumbersType) {
+    this.editorConfiguration.lineNumbers = lineNumbers;
+  }
+
+  setTabSize(tabSize: number) {
+    this.editorConfiguration.tabSize = tabSize;
+  }
+
+  enableMinimap(miniMap: Boolean) {
```

Comment:
We should ban the class primative types, please use `boolean`.

Reply:
ok, how can I throw null within primitive type? This method contains 3 possible results, true/false and removing minimap model at all (default option)

Reply:
`boolean | null`